runtime/autoload/ccomplete.lua

```diff
@@ -0,0 +1,892 @@
+----------------------------------------
+-- This file is generated via github.com/tjdevries/vim9jit
+-- For any bugs, please first consider reporting there.
+----------------------------------------
+
+local NVIM9 = require('_vim9script')
+local __VIM9_MODULE = {}
```

Comment:
any reason not to stick with the `M` convention?

Reply:
Conflicting variable names probably?

Reply:
I think I could make this do `M`, let me test that out.

Reply:
Should be fixed. If they happen to use `M`, it will do something like this:

```
  var M = true
  assert_equal(M, true)
```

to

```
    local __M__ = true
    NVIM9.fn.assert_equal(__M__, true)
```